recipe/activate.sh

```diff
@@ -1,10 +1,26 @@
 #!/bin/bash
-if [ "$(uname)" == "Darwin" ]
-then
-    # for Mac OSX
-    export KERAS_BACKEND=tensorflow
-elif [ "$(uname)" == "Linux" ]
-then
-    # for Linux
-    export KERAS_BACKEND=theano
+
+# get the location of the keras.json file
+KERAS_JSON="$HOME/.keras/keras.json"
+
+if [ -f $KERAS_JSON ]; then # if the file exists
+
+  BACKEND_KEY="backend" # value we are interested in finding
+  KEY_VALUE_REGEX="\"($BACKEND_KEY)\": \"([^\"]*)\"" # regex
+  JSON_TEXT=$(cat $KERAS_JSON) # data in keras.json file
+
+  if [[ $JSON_TEXT =~ $KEY_VALUE_REGEX ]]; then # check for a match
+    BACKEND_VALUE="${BASH_REMATCH[2]}" # get the user defined backend value
+    export KERAS_BACKEND=$BACKEND_VALUE # set the environmental variable
+  fi
```

Comment:
When a user activates the conda environment, all the activation scripts in the `anaconda/envs/<environment>/etc/conda/activate.d` are executed. The `keras_activate.sh` script then sets the `KERAS_BACKEND` environment variable which overrides the settings in the user's `.keras/keras.json` file, unless the user explicitly sets the `KERAS_BACKEND` variable before launching python and importing Keras. See my open issue #13